src/tools/cephfs/top/cephfs-top

```diff
@@ -0,0 +1,294 @@
+            x_coord_map[item] = (xp, nlen)
+            xp += nlen
+        self.topl.addstr(0, 0, ITEMS_PAD.join(heading), curses.A_STANDOUT | curses.A_BOLD)
+        return x_coord_map
+
+    def refresh_client(self, client_id, metrics, counters, client_meta, x_coord_map, y_coord):
+        for item in MAIN_WINDOW_TOP_LINE_ITEMS_END:
+            coord = x_coord_map[item]
+            if item == FS_TOP_MAIN_WINDOW_COL_ADDR_HOST:
+                self.mainw.addstr(y_coord, coord[0], f'{client_meta[CLIENT_METADATA_IP_KEY]}/'
+                                  f'{client_meta[CLIENT_METADATA_HOSTNAME_KEY]}')
+        for item in MAIN_WINDOW_TOP_LINE_ITEMS_START:
+            coord = x_coord_map[item]
+            if item == FS_TOP_MAIN_WINDOW_COL_CLIENT_ID:
+                self.mainw.addstr(y_coord, coord[0], client_id.split('.')[1])
+            elif item == FS_TOP_MAIN_WINDOW_COL_MNTPT:
+                self.mainw.addstr(y_coord, coord[0], client_meta[CLIENT_METADATA_MOUNT_POINT_KEY])
+            elif item == FS_TOP_MAIN_WINDOW_COL_MNT_ROOT:
+                self.mainw.addstr(y_coord, coord[0], client_meta[CLIENT_METADATA_MOUNT_ROOT_KEY])
+        cidx = 0
+        for item in counters:
+            coord = x_coord_map[item]
+            m = metrics[cidx]
+            typ = MAIN_WINDOW_TOP_LINE_METRICS[MGR_STATS_COUNTERS[cidx]]
+            if item.lower() in client_meta['valid_metrics']:
+                if typ == MetricType.METRIC_TYPE_PERCENTAGE:
+                    self.mainw.addstr(y_coord, coord[0], f'{calc_perc(m)}')
+                elif typ == MetricType.METRIC_TYPE_LATENCY:
+                    self.mainw.addstr(y_coord, coord[0], f'{calc_lat(m)}')
+            else:
+                self.mainw.addstr(y_coord, coord[0], "N/A")
+            cidx += 1
+
+    def refresh_clients(self, x_coord_map, stats_json):
+        counters = [m.upper() for m in stats_json[GLOBAL_COUNTERS_KEY]]
+        y_coord = 0
+        for client_id, metrics in stats_json[GLOBAL_METRICS_KEY].items():
+            self.refresh_client(client_id,
+                                metrics,
+                                counters,
+                                stats_json[CLIENT_METADATA_KEY][client_id],
+                                x_coord_map,
+                                y_coord)
+            y_coord += 1
+
+    def refresh_main_window(self, x_coord_map, stats_json):
+        self.refresh_clients(x_coord_map, stats_json)
+
+    def refresh_header(self, stats_json):
+        if not stats_json['version'] == FS_TOP_SUPPORTED_VER:
+            self.header.addstr(0, 0, 'perf stats version mismatch!')
+            return False
+        client_metadata = stats_json[CLIENT_METADATA_KEY]
+        num_clients = len(client_metadata)
+        num_mounts = len([client for client, metadata in client_metadata.items() if not
+                          metadata[CLIENT_METADATA_MOUNT_POINT_KEY] == 'N/A'])
+        now = datetime.now().strftime('%H:%M:%S')
```

Comment:
show full iso 8601 date, maybe?

Reply:
followed what top(1) shows by default ;)

But, you are right, we should probably switch.